tools/extensions/validate_extensions.py

```diff
@@ -0,0 +1,48 @@
+import json
```

Comment:
this has been added to prevent regression - should be removed before landing this PR